code/solutions/linting-and-testing/example_sqs_queue.yaml

```diff
@@ -0,0 +1,23 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example SQS queue (uksb-1q9p31idr)
+
+Resources:
+  MyQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      DelaySeconds: 900
+      Tags:
+        - Key: example_key_1
+          Value: example_value_1
+        - Key: example_key_2
+          Value: example_value_2
+
+Outputs:
+  QueueURL:
+    Description: The URL of your Amazon SQS Queue
```

Comment:
This SQS queue is created as part of the workshop, can we do better description here? Fro example: CloudFromation workshop SQS queue URL.